src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -95,6 +97,19 @@
   background-color: var(--pf-c-dropdown__toggle--BackgroundColor);
   border: none;
 
+  &.pf-m-disabled,
+  &:disabled {
+    pointer-events: none;
+
+    &:not(.pf-m-plain) {
+      --pf-c-dropdown__toggle--BackgroundColor: var(--pf-c-dropdown__toggle--disabled--BackgroundColor);
+
+      &::before {
+        border-color: var(--pf-c-dropdown__toggle--disabled--BackgroundColor);
```

Comment:
could you just do border: 0; to unset the border since technically it doesn't have a border?

Reply:
You can, I was following what the button component does, but now that i look at it again, the button uses `transparent` as the color. 

Seems like I should either use `border: 0;` or set `--pf-c-dropdown__toggle--BorderTopColor`, `--pf-c-dropdown__toggle--BorderRightColor`, etc to `transparent`, which allows a person to use a color if they want to. What do you think?

Reply:
AH thats a tricky one. I'm in favor of `border: 0` because the design doesn't have a border so why give people the opportunity to add a border when the design doesn't call for it.